charts/castai-cluster-controller/Chart.yaml

```
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-cluster-controller
 description: CAST AI cluster controller deployment chart.
 type: application
-version: 0.12.0
+version: 0.13.0
```

Sorry, 0.13.0 is already deployed 

```suggestion
version: 0.14.0
```